tests/integ/scala/test_datatype_suite.py

```diff
@@ -229,6 +294,170 @@ def test_dtypes(session):
     ]
 
 
+@pytest.mark.parametrize(
```

Comment:
should we also skip these tests in stored proc? Does stored proc connector have the corresponding change for struct type?

Reply:
By the time I merge this change it should have the corresponding change.